apps/frontend/src/public/.well-known/security.txt

```diff
@@ -0,0 +1,5 @@
+Contact: mailto:jai@modrinth.com
+Expires: 2030-01-01T00:00:00.000Z
```

Comment:
Does it work like ‘the current data is valid till x’? If yes, would it be too much? We can't say with 100% guarantee that Jai will remain the owner and security person for Modrinth until that time. This perhaps should be revised once a year.

Reply:
I believe it works more like a "the current data is invalid after x", not that it can't change before that, but I can't say this with certainty. My only concern with making it one year, would be forgetting to update it. It's tucked away in a folder that nobody visits, you know?